op-service/client/rpc.go

```diff
@@ -175,26 +175,31 @@ func IsURLAvailable(address string) bool {
 // It sets a timeout of 10s on CallContext & 20s on BatchCallContext made through it.
 type BaseRPCClient struct {
 	c               *rpc.Client
-	rpcTimeout      time.Duration
-	rpcBatchTimeout time.Duration
+	RpcTimeout      time.Duration
```

Comment:
Nit: For consistency with the data structure, and following go naming conventions, these should be `RPCTimeout` and `RPCBatchTimeout` (leaving the entire acronym capitalized)

Same goes for the `BaseRpcTimeout`

Reply:
Done